src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+
+    Animal.alive.push(this);
```

Comment:
не треба передавати health  в параметрах

```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;

    Animal.alive.push(this);
```